src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/JJDisputeService.java

```diff
@@ -279,7 +281,7 @@ public JJDispute setStatus(String ticketNumber, JJDisputeStatus jjDisputeStatus,
 		YesNo seizedYn = courtAppearance != null ? courtAppearance.getJjSeized() : null;
 		JJDisputeCourtAppearanceAPP aattCd = courtAppearance != null ? courtAppearance.getAppCd() : null;
 		JJDisputeCourtAppearanceDATT dattCd = courtAppearance != null ? courtAppearance.getDattCd() : null;
-		String staffPartId = null; // TODO: Figure out mapping for staffPartId - is it the same partId??
+		String staffPartId = principal.getName(); // StaffId comes from the person currently logged in, the Principal
```

Comment:
try CustomUserDetails user = (CustomUserDetails) SecurityContextHolder.getContext().getAuthentication();
		String staffPartId = user.getPartId();

Reply:
Done. BTW, this looks like a bug. StaffPartId is always "System" ... I don't think it is properly set by staff-api. (Should it be the part-id pulled from keycloak?)

Here's a screenshot of this object at this line in the code.
![image](https://user-images.githubusercontent.com/55215368/220993729-0cf23beb-2ce1-453c-87fd-2ff0e48b92f1.png)